Documentation/sources/content-card-ui/public-classes/Styles/smallimageuistyle.md

```diff
@@ -0,0 +1,36 @@
+# Class - SmallImageUIStyle
+
+Class representing the style for a small image AEP UI.
+
+## Public Properties
+
+| Property               | Type                                                         | Description                                                  |
+| ---------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| cardStyle              | [AepCardStyle](./aepcardstyle.md)                            | The style for the card.                                      |
+| rootRowStyle           | [AepRowStyle](./aeprowstyle.md)                              | The style for the root row.                                  |
+| imageStyle             | [AepImageStyle](./aepimagestyle,md)                          | The style for the image.                                     |
+| textColumnStyle        | [AepColumnStyle](./aepcolumnstyle.md)                        | The style for the column containing the title, body and buttons. |
+| titleTextStyle         | [AepTextStyle](./aeptextstyle.md)                            | The text style for the title.                                |
+| bodyTextStyle          | [AepTextStyle](./aeptextstyle.md)                            | The text style for the body.                                 |
+| buttonRowStyle         | [AepRowStyle](./aeprowstyle.md)                              | The style for the row containing the buttons.                |
+| buttonStyle            | Array<Pair<[AepButtonStyle](./aepbuttonstyle.md), [AepTextStyle](./aeptextstyle.md)>> | The style for the buttons.                                   |
```

Comment:
@spoorthipujariadobe  - I have second thoughts about how the api for button styling is structured 😅 .
Wondering if we should tweak the implementation to Allow AepButtonStyle to accept an AepTextStyle. That way this is simpler. Furthermore,  it will allow us to define an IconStyle for the same button without changing the API signature of the SmallImageUI later if/when we add Icon capability to buttons.

Reply:
Sounds good. I will have it in the next PR with the tests